packages/gcc10.rb

```diff
@@ -58,7 +66,7 @@ def self.build
     sed -i '77a #ifndef PATH_MAX' libsanitizer/asan/asan_linux.cpp)"
     Dir.chdir("objdir") do
       case ARCH
-      when 'armv7l', 'aarch64'
+      when 'armv7l', 'armv8l', 'aarch64'
```

Comment:
Is armv8l a real architecture?  I know of no Chrome(book/box) hardware that has this architecture.

Reply:
That's there solely for the situation where it is building virtualized for building.

The raspberry pi is going to end up being the cheapest arm Chromebox at some point, right?

Reply:
Doesn't `lib/const.rb` already have the logic to switch armv8l over to armv7l?